pkg/core/native_contract_test.go

```diff
@@ -95,6 +126,38 @@ func (tn *testNative) sum(_ *interop.Context, args []stackitem.Item) stackitem.I
 	return stackitem.NewBigInteger(s1.Add(s1, s2))
 }
 
+func (tn *testNative) callOtherContractWithoutArgs(ic *interop.Context, args []stackitem.Item) stackitem.Item {
+	vm := ic.VM
+	vm.Estack().PushVal(stackitem.NewArray([]stackitem.Item{})) // no args
+	vm.Estack().PushVal(args[1])                                // method
+	vm.Estack().PushVal(args[0])                                // contract hash
+	err := contractCall(ic)
+	if err != nil {
+		return stackitem.NewBigInteger(big.NewInt(-1))
+	}
+	_ = vm.Run()
+	if vm.HasFailed() {
+		return stackitem.NewBigInteger(big.NewInt(-2))
```

Comment:
It's a bit strange that we're returning something here, but I guess there are not that many options we have.

Reply:
It was a test method, so I just put something there. What should we do in this case?

Reply:
Well, the vm already has failed. We shouldn't touch it, but at the same time we have to return something for `Func`, so it probably could be just about anything.